packages/cli/doc-core/src/theme-default/layout/DocLayout/index.module.scss

```diff
@@ -54,13 +54,13 @@
 
 @media (min-width: 1440px) {
   .docLayout {
-    width: 1440px;
+    width: calc(1440px - 100vw + 100%);
```

Comment:
Why `-100vw + 100%`?

Reply:
100vw is viewport ，100% is body width  ，the rest is the width of the scroll bar